src/main/java/nvt/kts/ticketapp/controller/event/EventController.java

```diff
@@ -57,6 +49,8 @@
 
     @Autowired
     private LocationService locationService;
+    @Autowired
+    private TicketService ticketService;
```

Comment:
jel treba ticket service ovde, ne koristi se?

Reply:
Bravo brate, skroz sam zaboravio to da obrisem. Nesto sam tetirao pa pravio endpoint da direktno gadjam taj servis i ostao mi autowired. obrisacu pa kad prodje sve spojiti